src/services.py

```
@@ -50,31 +45,17 @@ def send_answer(message: "Message", user: "UsersOrm", answer: str) -> None:
-        translation = translate(answer, target_language=user.target_language)
-        translation_md = markdownify(translation)
-        if len(translation_md) > 4096:  # noqa: PLR2004
-            chunks = smart_split(translation, 4000)
-            for text in chunks:
-                text_md = markdownify(text)
-                bot.reply_to(message, text_md, parse_mode="MarkdownV2")
-                time.sleep(1)
-        else:
-            bot.reply_to(message, translation_md, parse_mode="MarkdownV2")
+    answer_md = markdownify(answer)
+    if len(answer_md) > 4000:  # 4096 limit # noqa: PLR2004
```

Ruff (0.14.5)

Unused `noqa` directive (non-enabled: `PLR2004`)

Remove unused `noqa` directive

(RUF100)